crates/core/app/src/server/consensus.rs

```diff
@@ -25,7 +25,10 @@ fn trace_events(events: &[Event]) {
         let span = tracing::debug_span!("event", kind = ?event.kind);
         span.in_scope(|| {
             for attr in &event.attributes {
-                tracing::debug!(k = ?attr.key, v=?attr.value);
+                tracing::debug!(
```

Comment:
These seemed to be reasonable. Tendermint no longer guarantees that keys and values are utf8: https://docs.rs/tendermint/0.40.0/tendermint/abci/enum.EventAttribute.html